fs/fs.go

```diff
@@ -455,6 +461,58 @@ func (fs *filesystem) getSociContext(ctx context.Context, imageRef, indexDigest,
 	return c, err
 }
 
+func getIDMappedMountpoint(mountpoint, activeLayerKey string) string {
```

Comment:
nit: can we rename this to `activeLayerId`?

`key` has a specific meaning to containerd and that's not what we mean.